src/RenderWebGL.js

```diff
@@ -1002,10 +1002,19 @@ class RenderWebGL extends EventEmitter {
      * RenderConstants.ID_NONE if there is no Drawable at that location.
      */
     pick (centerX, centerY, touchWidth, touchHeight, candidateIDs) {
+        const bounds = this.clientSpaceToScratchBounds(centerX, centerY, touchWidth, touchHeight);
+
         candidateIDs = (candidateIDs || this._drawList).filter(id => {
             const drawable = this._allDrawables[id];
+            const drawableBounds = this.getBounds(id);
```

Comment:
The `getBounds` method updates the convex hull on the target drawable, which is a very slow operation.
Maybe use `drawable.getFastBounds` instead?

Reply:
I used amend push